WordPress/WordPressTest/PostTests.swift

```diff
@@ -0,0 +1,168 @@
+import Foundation
+import XCTest
+
+@testable import WordPress
+
+class PostTests: XCTestCase {
+
+    private let context: NSManagedObjectContext = {
+        let context = NSManagedObjectContext(concurrencyType: .MainQueueConcurrencyType)
+        context.parentContext = TestContextManager.sharedInstance().mainContext
+
+        return context
+    }()
+
+    private func newTestBlog() -> Blog {
+        return NSEntityDescription.insertNewObjectForEntityForName(BlogEntityName, inManagedObjectContext: context) as! Blog
+    }
+
+    private func newTestPost() -> Post {
+        return NSEntityDescription.insertNewObjectForEntityForName(PostEntityName, inManagedObjectContext: context) as! Post
+    }
+
+    private func newTestPostCategory() -> PostCategory {
+        return NSEntityDescription.insertNewObjectForEntityForName(PostCategoryEntityName, inManagedObjectContext: context) as! PostCategory
+    }
+
+    private func newTestPostCategory(name: String) -> PostCategory {
+        let category = newTestPostCategory()
+        category.categoryName = name
+
+        return category
+    }
+
+    override func setUp() {
+        super.setUp()
+    }
+
+    override func tearDown() {
+        context.rollback()
+        super.tearDown()
+    }
+
+    func testThatNoCategoriesReturnEmptyStringWhenCallingCategoriesText() {
+        let post = newTestPost()
+        let categoriesText = post.categoriesText()
+
+        XCTAssert(categoriesText == "")
+    }
+
+    func testThatSomeCategoriesReturnAListWhenCallingCategoriesText() {
+
+        let post = newTestPost()
+
+        post.categories = Set<PostCategory>(arrayLiteral: newTestPostCategory("1"), newTestPostCategory("2"), newTestPostCategory("3"))
```

Comment:
That's actually an array literal (ie: it's not a set until you assign it to a `Set` object).  In fact if you do `let myConst = ["a", "b", "c"]`, you'll get an array.

That said, your comment is still valid for this case as it makes initialization less verbose, so I'll take it.  Thanks!